deployment/makefiles/scripts/configurations.json

```diff
@@ -0,0 +1,88 @@
+[
+  {
+    "Classification": "spark",
+    "Properties": {
+      "maximizeResourceAllocation": "true"
+    }
+  },
+  {
+    "Classification": "spark-defaults",
+    "Properties": {
+        "spark.driver.maxResultSize": "3G",
+        "spark.dynamicAllocation.enabled": "true",
+        "spark.shuffle.service.enabled": "true",
+        "spark.shuffle.compress": "true",
+        "spark.shuffle.spill.compress": "true",
+        "spark.rdd.compress": "true",
+        "spark.yarn.executor.memoryOverhead": "1G",
+        "spark.yarn.driver.memoryOverhead": "1G",
+        "spark.driver.maxResultSize": "3G",
+        "spark.executor.extraJavaOptions" : "-XX:+UseParallelGC -Dgeotrellis.s3.threads.rdd.write=64"
```

Comment:
What's the default for `geotrellis.s3.threads.rdd.write`? (I'm assuming that this speeds up vector tile writes but doesn't impact ORC writes.)

Reply:
TBH, I'm not sure where in GTs configs we actually set the value.  It's nested down somewhere in the guts, or is provided by the typesafe config library.  We do set this value to improve performance, but it does lead to occasional spark task failures from cramming too much data into S3.  Not clear if lowering this value would clear the errors without lowering overall performance.

Reply:
Huh.

Longer-term, I'd like to investigate writing zips of vector tile pyramids to S3 (a preliminary implementation of this was in the Zeppelin notebook I sent over), since S3 writes can get expensive, both in terms of latency and $$$. I'll open an issue for discussing that.